### DISCUSSION TITLE
versionCode & versionName

### DISCUSSION THREAD

**Discussion by mizowgame** [abdellatif-laghjaj/android-all-in-one-video-downloader#2] [Q&A] [answered]
hi thank you for the project it's amazing but how to change versionCode & versionName :)

**Comment by abdellatif-laghjaj** [comment, marked as answer]
Hello, sorry for the **late** response I didn't **receive** a notification of your **question**!!
But here's how you can change the **version code** and the **version name:**
Go to `build.gradle (Project)` and change them.

![ScreenShot_20221109212650](https://user-images.githubusercontent.com/79521157/200935057-2a912fa2-701a-4cfb-a649-c1640fb5edef.png)
